tests/helpers/expect-imports.js

```diff
@@ -0,0 +1,9 @@
+const exclude = [
```

Comment:
this can use ember-macro-test-helpers/expect-imports now

Reply:
@kellyselden updated PR to use `expect-imports` from `ember-macro-test-helpers`.